openapi/platform/platform.yaml

```diff
@@ -303,6 +303,11 @@ paths:
           in: query
           name: sort
           description: Sort busines location based on selected option. Defaulted to -updatedAt (Desc updatedAt)
+        - schema:
+            type: boolean
+          in: query
+          name: isNewCategory
+          description: 'isNewCategory based on value true or false gives Business Categories.For new categories, value of isNewCategory should be true but for old categories value of isNewCategory should be false.'
```

Comment:
You may describe it in this way as well,
isNewCategory is of type boolean which returns new/old business categories. value of isNewCategory true return new business categories and value of isNewCategory false return old business categories.